redact/tools/v4.py

```diff
@@ -98,6 +98,18 @@ def redact_file(
         True, help="Specify whether to automatically delete the job from the backend"
     ),
     verbose_logging: bool = typer.Option(False, help="Enable very noisy logging."),
+    areas_of_interest: Optional[str] = typer.Option(
+        None,
+        help=(
+            "__Experimental__ Areas of interest's left corner coordinates x and y, their height and width. Must be "
+            "provided as a string in the following format [x,y,width,height]. Multiple areas could be added if needed. "
+            "For example, "
+            "'...&areas_of_interest=[0,0,960,540]', '...&areas_of_interest=[[0,0,960,540],[0,540,960,540]]', "
```

Comment:
I think this sample doesn't make sense for a cli. It looks more like the format you would use in a url.

Reply:
I would expect the parameter to work like this

`redact-file --areas_of_interest 0,0,960,540 0,0,960,540 0,0,960,540`

Reply:
updated